src/main/kotlin/presentation/utils/UserInputReaderImpl.kt

```diff
@@ -0,0 +1,20 @@
+package presentation.utils
+
+class UserInputReaderImpl: UserInputReader {
+    override fun getUserInput(message: String): String {
```

Comment:
I think here instead of name message maybe - input or if just one charachter you can yes char